### PR TITLE
feat(frontend) rewording of 'Publications' in 'News'

### PR DESCRIPTION
**Pull request by charline-rageade**
#4950 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

**Comment by charline-rageade** [comment]
It's reworking, unit and functional test aren't necessary.

**Comment by PhilippeAccorsi** [comment]
I'm not sure but I think we have done some modification for yarn in 3.9.2 (because yarn version change and something goes wrong on oldest installation).

So to build a new docker images, we need the yarn fix addes in 3.9.2 (https://github.com/tracim/tracim/issues/4830). Please see that with @grignards I think to seen what can be done for branch `maintenance/3.8`

**Comment by PhilippeAccorsi** [comment]
@charline-rageade

**Comment by charline-rageade** [comment]
thanks @PhilippeAccorsi, I found it strange that my tests went local and the CI failed. I'll check with Sebastien.

**Comment by PhilippeAccorsi** [comment]
> I found it strange that my tests ran locally and the CI failed.

If you try to install tracim from scratch (tracim and his tool), you can seen same error. Yarn install process is broken.

**Comment by PhilippeAccorsi** [comment]
You need to commit someting if you want to run ci. The last commit have `[skip ci]` :)

**Comment by PhilippeAccorsi** [comment]
@charline-rageade

**Comment by PhilippeAccorsi** [comment]
When all ready to merge, please add explicit comment about this fix and the fix of yarn.

File: frontend_app_workspace_advanced/i18next.scanner/de/translation.json [hunk 1]
```diff
@@ -51,8 +51,8 @@
...
   "There are no requests yet.": "Es sind noch keine Anfragen vorhanden.",
   "This is the role that members will have by default when they join your space (for open and on request spaces only).": "Dies ist die Rolle, die Mitglieder standardmäßig haben, wenn sie Ihrem Bereich beitreten (nur für offene Bereiche und Bereiche auf Anfrage).",
-  "Publications activated": "Publikationen aktiviert",
-  "Publications deactivated": "Publikationen deaktiviert",
-  "Error while activating publications": "Fehler beim Aktivieren von Publikationen",
-  "Error while deactivating publications": "Fehler beim Deaktivieren von Publikationen"
+  "News activated": "Nachrichten aktiviert",
+  "News deactivated": "Nachrichten deaktiviert",
+  "Error while activating news": "Fehler beim Aktivieren von Nachrichten",
+  "Error while deactivating news": "Fehler beim Dektivieren von Nachrichten"
```
Comment (MariePrandstetter) [inline frontend_app_workspace_advanced/i18next.scanner/de/translation.json:57]: Fehler beim Deaktivieren von Nachrichten

**Review by MariePrandstetter — APPROVED** [review]

File: frontend_app_workspace_advanced/i18next.scanner/de/translation.json (hunk 1 above)
Comment (charline-rageade) [reply frontend_app_workspace_advanced/i18next.scanner/de/translation.json]: Thanks it's fixed :)

**Comment by charline-rageade** [comment]
> I'm not sure but I think we have done some modification for yarn in 3.9.2 (because yarn version change and something goes wrong on oldest installation).
So to build a new docker images, we need the yarn fix addes in 3.9.2 (#4830). Please see that with @grignards I think to seen what can be done for branch maintenance/3.8

To solve the problem encountered with the version of yarn on the maintenance 3.8 branch. 
It was necessary to cherry-pick : 
-  b1afaf04924ecd749c848907f54febfcfa491eea
-  cf1e9131b0fa96d2916ba9282e29cd8bbf6a8a08

**Review by MariePrandstetter — APPROVED** [review]

**Comment by Skylsmoi** [comment]
manually tested

**Pull request merged**